melodies_monet/plots/surfplots.py

```diff
@@ -106,12 +109,34 @@ def new_color_map():
         Orange and blue color map
         
     """
-    top = mpl.cm.get_cmap('Blues_r', 128)
-    bottom = mpl.cm.get_cmap('Oranges', 128)
+    top = mpl.cm.get_cmap("Blues_r", 128)
+    bottom = mpl.cm.get_cmap("Oranges", 128)
```

Comment:
```suggestion
    top = plt.get_cmap("Blues_r", 128)
    bottom = plt.get_cmap("Oranges", 128)
```

From [the notes](https://matplotlib.org/stable/users/prev_whats_new/whats_new_3.5.0.html#colormap-registry-experimental)

> `plt.get_cmap()` and `plt.register_cmap()` will continue to be supported for backward compatibility.